sp1/src/io.rs

```diff
@@ -125,10 +125,10 @@ pub mod proof_serde {
 
     #[cfg(test)]
     mod tests {
-        use crate::{
-            utils::{tests::FIBONACCI_IO_ELF, BabyBearBlake3},
-            SP1ProofWithIO, SP1Prover, SP1Stdin, SP1Verifier,
-        };
+        use crate::{utils::BabyBearBlake3, SP1ProofWithIO, SP1Prover, SP1Stdin, SP1Verifier};
+
+        pub const FIBONACCI_IO_ELF: &[u8] =
```

Comment:
Note: Didn't want to re-export all of the ELF's from `programs::tests` as it would slow down compilation time (would force `sp1-core` to compile all of the ELF's before `sp1` could import). Instead, just include the only ELF required by the tests in `io.rs`